### DISCUSSION TITLE
Heal delay

### DISCUSSION THREAD

**Discussion by Abkim27** [FakeFishGames/Barotrauma#9952] [Ideas and suggestions]
Make healing some kind of timed action, so you need to be in safe place, where no one will interrupt you, to heal yourself.
After using opium or anything else on yourself or someone you will have a delay and after 1-2 seconds opium actually uses. If someone stun you in that delay, opium doesn't use.
So that change will make barotrauma more strategical, because you cannot heal in battle and make medic far more vulnareble, because now they can heal yourself even when you shoot him in the face.

**Comment by Ek-Videogames** [comment]
The general idea has been tossed around a few times before.

One consideration that will be needed is making time-to-kill slower from blood loss & unconscious death timer so 'long duration treatments' are able to complete before the afflictions being treated end up being lethal. Even moreso if requiring a medic.

**Reply by Abkim27** [reply]
That has no need, because you can use blood bag first.
